### DISCUSSION TITLE
Discord login seems broken

### DISCUSSION THREAD

**Discussion by peterhinch** [micropython/micropython#15878] [General]
Any one else having trouble logging in? It's totally borked here :(

**Comment by robert-hh** [comment]
Discord is working here.

**Comment by peterhinch** [comment]
I enter username and password as usual, then it takes me through two captcha tests (which it never did before). But then I'm back at the login screen where it again presents me with the "Wait! Are you human?" dialog with infinite recursion.

**Reply by robert-hh** [reply]
I logged out and then I ran into the same problem when trying to log in with the user name. What worked was using my e-mail address registered at discord instead of the user name with the same password.

**Reply by peterhinch** [reply]
Trouble is, I always log in with my email address :(

This seems to happen when organisations change their security settings. I use a Yubikey to log in to GitHub - for a while this stooped working properly and I had to use Google Authenicator. Then the Yubikey started working again for no obvious reason. Perhaps Discord will recover, or support modern authentication rather than passwords and Captchas.

**Reply by peterhinch** [reply]
In case anyone else encounters this, I've fixed it. I had received an email from Discord - it claimed I had logged in from a different location. Since this was untrue, I'd ignored it. Confirming registration fixed it.

**Very** confusing behaviour.
